app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,50 @@
+module Api
+  module V1
+    # User Controller
+    class UsersController < ApplicationController
+      before_action :set_user, only: %i[show update destroy]
+
+      # GET /users/1
+      def show
+        render json: User.find(params[:id])
```

Comment:
you already have the user loaded in @user variable, since you are using a before_action filter which executes before each one